creator-node/test/pollingTracks.test.js

```diff
@@ -514,6 +515,16 @@ describe('test Polling Tracks with mocked IPFS', function () {
       ]
     })
 
+    // Requests with too many wallet keys should be rejected
+    const largeWalletPublicKeys = Array.from(
+      { length: MAX_BATCH_CLOCK_STATUS_BATCH_SIZE + 1 },
+      (_, i) => i + 'a'
+    )
+    resp = await request(app)
+      .post(`/users/batch_clock_status`)
+      .send({ walletPublicKeys: largeWalletPublicKeys })
+      .expect(400)
```

Comment:
nice!!

Reply:
there's a way to assert on the exact error msg here as well which we should do here

ex https://github.com/AudiusProject/audius-protocol/blob/master/creator-node/test/audiusUsers.test.js#L254